content/ch-quantum-hardware/calibrating-qubits-pulse.ipynb

```diff
@@ -1135,10 +1256,23 @@
     "    drive_chan = pulse.drive_channel(qubit)\n",
     "    pulse.set_frequency(rough_qubit_frequency, drive_chan)\n",
     "    pulse.call(pi_pulse)\n",
-    "    pulse.delay(delay, drive_chan)\n",
-    "    pulse.measure(qubits=[qubit], registers=[pulse.MemorySlot(mem_slot)])\n",
+    "    pulse.delay(delay, drive_chan)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "qc_t1 = QuantumCircuit(1, 1)\n",
     "\n",
-    "t1_schedules = [t1_schedule.assign_parameters({delay: get_dt_from(d)}, inplace=False) for d in delay_times_sec]"
+    "qc_t1.x(0)\n",
+    "qc_t1.delay(delay, 0)\n",
+    "qc_t1.measure(0, 0)\n",
+    "qc_t1.add_calibration(\"x\", (0,), t1_schedule, [delay])\n",
```

Comment:
Here you attached `t1_schedule` to `x` gate. This is a bug, because `t1_schedule` contains delay while the program written in circuit also contains the same delay, so you have 2 delays in total. What you can do here is
```python
qc_t1.add_calibration("x", (0,), pi_pulse)
```